src/xrGame/quadtree_inline.h

```diff
@@ -102,6 +108,9 @@ IC void CSQuadTree::insert(_object_type* object)
         if (!*node)
             *node = m_nodes->get_object();
 
+        if (!node)
```

Comment:
So, [back to the old discussion](https://github.com/OpenXRay/xray-16/pull/1432#discussion_r1306691967), we can't do `*node` (dereference) two lines above if `node` is `nullptr`, so this `if (!node)` check is actually **late** and we need to check for nullptr earlier.